File: src/Omnia.ProcessManagement.Models/ShapeTemplates/ShapeTemplatePentagonSettings.cs
using Omnia.ProcessManagement.Models.FabricShape;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ShapeTemplates
{
    public class ShapeTemplatePentagonSettings : ShapeTemplateSettings
    {
        public double ArrowWidthPercent { get; set; }
        public double ArrowHeightPercent { get; set; }
        public bool IsLine { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/shapetemplates/ShapeTemplatePentagonSettings.ts
import { ShapeTemplateSettings } from './ShapeTemplateSettings';
import { FabricShapeData } from '../../../processshape/fabricshape/FabricShapeData';
import { ShapeTemplateType } from '../enums/Enums';

export interface ShapeTemplatePentagonSettings extends ShapeTemplateSettings {
    type: ShapeTemplateType.PentagonShape,
    arrowWidthPercent: number;
    arrowHeightPercent: number;
    isLine: boolean;
}